tests/test_page.py

```diff
@@ -833,10 +833,14 @@ def test_empyt_password_1088():
     len(reader.pages)
 
 
-@pytest.mark.xfail(reason="#1088 / #1126")
+@pytest.mark.external
+# @pytest.mark.xfail(reason="#1088 / #1126")
 def test_arab_text_extraction():
-    reader = PdfReader(SAMPLE_ROOT / "015-arabic/habibi.pdf")
```

Comment:
I would have proposed to keep the odd habibi.pdf fixing with the expectation and just add yours no problem to keep it in the same test.

Reply:
Since the odd habibi.pdf has a ligature "h" that is composed of Arabic+Latin+White characters, I think it's better to move it into a separate test so we can tackle it the ligature decomposition PR.

What are your thoughts?

Also, think that #1088 reason is wrong and should be #1188.

Reply:
I don't mind splitting in two test however I think Habibi (legacy) assertion should work as expected. Any ideas why it is currently not working
About xfail I hope we will be able to get rid of it

Reply:
@pubpub-zz 
The reason why Habibi Legacy isn't working is explained in [this comment](https://github.com/py-pdf/pypdf/pull/1597#discussion_r1099985305) below following your question!